packages/scandipwa/src/component/ProductTab/ProductTab.component.tsx

```diff
@@ -12,33 +12,23 @@
 import { PureComponent } from 'react';
 
 import { ReactElement } from 'Type/Common.type';
-import { noopFn } from 'Util/Common';
 
 import { ProductTabComponentProps } from './ProductTab.type';
 
 import './ProductTab.style';
 
 /** @namespace Component/ProductTab/Component */
 export class ProductTabComponent extends PureComponent<ProductTabComponentProps> {
-    static defaultProps: Partial<ProductTabComponentProps> = {
-        onClick: noopFn,
-        isActive: false,
-    };
-
-    __construct(props: ProductTabComponentProps): void {
-        super.__construct?.(props);
-
-        this.onClick = this.onClick.bind(this);
-    }
+    onClick = () => {
+        const { onClick, tab: { id } } = this.props;
 
-    onClick(): void {
-        const { onClick, tabName } = this.props;
-
-        onClick(tabName);
-    }
+        onClick(id);
+    };
```

Comment:
How are you gonna be a plugin to it? 
As I know arrow functions not working for plugins